elyses-enchantments/enchantments.js

```diff
@@ -22,8 +22,9 @@ export function getItem(cards, position) {
  * @returns {number[]} the cards with the change applied
  */
 export function setItem(cards, position, replacementCard) {
-  cards[position] = replacementCard;
-  return cards;
+  let newCards = Array.from(cards);
```

Comment:
We can use the _spread_ [syntax](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Spread_syntax) when cloning arrays.

```js
  let newCards = [ ...cards ];
```

Reply:
@gomain fixed.